sbtc-core/src/signer/mod.rs

```diff
@@ -0,0 +1,116 @@
+/// An sBTC signer configuration
+pub mod config;
+
+use std::collections::HashMap;
+
+use crate::SBTCError;
+use crate::{signer::config::Config, SBTCResult};
+use bitcoin::{Network, PrivateKey, PublicKey, Transaction as BitcoinTransaction};
+use p256k1::ecdsa;
+use url::Url;
+use wsts::{bip340::SchnorrProof, common::Signature};
+
+#[derive(Default, Clone, Debug)]
+/// Signers' data required for weighted distributed signing
+pub struct Signers {
+    /// Signer id to public key mapping
+    pub signer_ids: HashMap<u32, ecdsa::PublicKey>,
+    /// Vote share to public key mapping
+    pub vote_shares: HashMap<u32, ecdsa::PublicKey>,
+}
+
+/// sBTC Signer trait
+pub trait SBTCSigner {
```

Comment:
This trait is again just returning public keys, so the name doesn't seem particularly descriptive.  So I'd probably call the previous trait `SignerPublicKeys`, and this trait just `PublicKeys` (or `SBTCPublicKeys` if you really want the extra token).